### PR TITLE
Feat: add dropshadows

### PR DESCRIPTION
**Pull request by mairin**
### What does this PR do?

Adds dropshadows to the main nav, the settings nav, and an inner shadow highlight to the top of the window just under the titlebar.

### Screenshot/screencast of this PR

Screenshot from 2023-05-01 20-36-33
Screenshot from 2023-05-01 20-36-23
Screenshot from 2023-05-01 20-36-17

### What issues does this PR fix or reference?

supports https://github.com/containers/podman-desktop/issues/2287

### How to test this PR?

Try the main nav, the settings nav, and various form dialogs... make sure the drop shadows show up along the left side, particularly on long-scrolling screens.

### CODE REVIEW

File: packages/renderer/src/App.svelte [hunk 1]
```diff
@@ -70,8 +70,11 @@ window.events?.receive('display-help', () => {
 </script>
 
 <Route path="/*" breadcrumb="Home" let:meta>
-  <main class="min-h-screen flex flex-col h-screen bg-zinc-900">
-    <header id="navbar" class="text-gray-700 bg-zinc-900 body-font" style="-webkit-app-region: drag;">
+  <main class="min-h-screen flex flex-col h-screen bg-charcoal-800">
+    <header
+      id="navbar"
+      class="text-gray-700 bg-charcoal-900 body-font shadow-titlebar z-999;"
```
Comment (benoitf) [inline packages/renderer/src/App.svelte:76]: do we need the z-index for the shadows ?

Comment (benoitf) [reply packages/renderer/src/App.svelte:76]: @mairin ?

Comment (mairin) [reply packages/renderer/src/App.svelte:76]: I'm checking, I think I needed it but let me double check

Comment (mairin) [reply packages/renderer/src/App.svelte:76]: Yeh, it's definitely needed. Without raising the z-index, you can't see the highlight line along the screen at the bottom of the titlebar. Now, whether or not it needs to be 999 specifically is debatable 😃

**Review by benoitf — APPROVED** [review]

**Review by cdrage — APPROVED** [review]

**Pull request merged**